snuba/consumers/consumer_builder.py

```diff
@@ -84,13 +88,7 @@ def __init__(
 
         # XXX: This can result in a producer being built in cases where it's
         # not actually required.
-        self.producer = Producer(
-            {
-                "bootstrap.servers": ",".join(self.bootstrap_servers),
-                "partitioner": "consistent",
-                "message.max.bytes": 50000000,  # 50MB, default is 1MB
-            }
-        )
+        self.producer = Producer(self.broker_config)
```

Comment:
This will fail to apply the producer settings (partitioner and message.max.bytes) since those are applied by `build_kafka_producer_configuration` while`self.broker_config` is produced by `get_default_kafka_configuration`.
Please create two separate broker configs above in the constructor. One for consumers and the other for producers.

Reply:
you are right, I missed it. fixed now